src/EnergyPlus/DataSizing.hh

```diff
@@ -782,33 +791,32 @@ namespace DataSizing {
         Real64 FractionOfAutosizedHeatingAirflow; // fraction of of heating supply air flow rate an airloop
         Real64 FlowPerCoolingCapacity;            // ratio of cooling supply air flow rate to cooling capacity of an airloop
         Real64 FlowPerHeatingCapacity;            // ratio of heating supply air flow rate to heating capacity of an airloop
-        int CoolingPeakLoadType;                  // Type of peak to size cooling coils on   1=SensibleCoolingLoad; 2=TotalCoolingLoad
+        PeakLoad coolingPeakLoad;                 // Type of peak to size cooling coils on SensibleCooling or TotalCooling
         CapacityControl CoolCapControl;           // type of control of cooling coil  VAV, Bypass, VT, OnOff
         Real64 OccupantDiversity;                 // occupant diversity
 
         // Default Constructor
         SystemSizingInputData()
-            : AirLoopNum(0), LoadSizeType(0), SizingOption(0), CoolOAOption(0), HeatOAOption(0), DesOutAirVolFlow(0.0), SysAirMinFlowRat(0.0),
-              SysAirMinFlowRatWasAutoSized(false), PreheatTemp(0.0), PrecoolTemp(0.0), PreheatHumRat(0.0), PrecoolHumRat(0.0), CoolSupTemp(0.0),
-              HeatSupTemp(0.0), CoolSupHumRat(0.0), HeatSupHumRat(0.0), CoolAirDesMethod(0), DesCoolAirFlow(0.0), HeatAirDesMethod(0),
-              DesHeatAirFlow(0.0), ScaleCoolSAFMethod(0), ScaleHeatSAFMethod(0), SystemOAMethod(SysOAMethod::Invalid), MaxZoneOAFraction(0.0),
-              OAAutoSized(false), CoolingCapMethod(0), HeatingCapMethod(0), ScaledCoolingCapacity(0.0), ScaledHeatingCapacity(0.0),
-              FloorAreaOnAirLoopCooled(0.0), FloorAreaOnAirLoopHeated(0.0), FlowPerFloorAreaCooled(0.0), FlowPerFloorAreaHeated(0.0),
-              FractionOfAutosizedCoolingAirflow(1.0), FractionOfAutosizedHeatingAirflow(1.0), FlowPerCoolingCapacity(0.0),
-              FlowPerHeatingCapacity(0.0), CoolingPeakLoadType(0), // wfb
-              CoolCapControl(CapacityControl::Invalid)             // wfb
+            : AirLoopNum(0), loadSizingType(LoadSizing::Invalid), SizingOption(0), CoolOAOption(0), HeatOAOption(0), DesOutAirVolFlow(0.0),
```

Comment:
In general, if you are ever touching a default constructor like this, try to just delete it and move the initialization to the variables themselves.  No big deal for now.